### DISCUSSION TITLE
Consistency between b-parasites

### DISCUSSION THREAD

**Discussion by jrhbcn** [rbaron/b-parasite#141] [General]
Hello,

I am experiencing some variability on the measuring between my b-parasites. I have 5 of them and all seem to read different values. To test them, I have put all of them in the same pot and I got different humidity values that does not seem very consistent. 

This is my setup now:

![image](https://github.com/rbaron/b-parasite/assets/22198270/8edcd7c3-b343-4fa8-a68d-0e529fa4f8f6)

And this is the readings in the last 48 hours:

<img width="1444" alt="image" src="https://github.com/rbaron/b-parasite/assets/22198270/04e9661f-68bb-4580-8fea-ea683c967b07">

As you can see, there is a difference of 24 points between sensors in the same pot.  And these (sensors 3 and 4) are just one next to the other.

Is this normal? Is there something I can do to improve the consistency of the sensors?

**Comment by rbaron** [comment]
Hi @jrhbcn,

I find that the sensors are sensitive to how well they are "set" in place. See for example #104 and my comment there. My first suggestion would be to water the pot a few times so they all make more or less equal "contact" with the soil. 

Also, are the boards coated? We've seen some uncoated boards drift issues [(wiki)](https://github.com/rbaron/b-parasite/wiki/Conformal-Coating#why-coat-the-boards), and I imagine coating can also introduce some variability in the readings depending on the process.
